AdminApi.Sdk/Endpoints/Identities/Types/CreateIdentityRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Backbone.AdminApi.Sdk.Endpoints.Identities.Types;
+
+public class CreateIdentityRequest
+{
+    public required string ClientId { get; set; }
+    public required string ClientSecret { get; set; }
+    public required byte[] IdentityPublicKey { get; set; }
+    public required string DevicePassword { get; set; }
+    public required byte IdentityVersion { get; set; }
+    public required CreateIdentityRequestSignedChallenge SignedChallenge { get; set; }
```

Comment:
Can you remove this property from the SDK, as well as from the backend code please? In the IdentitiesController of the backend you then have to use some default values. Just use some dummy values:


```c#
public async Task<IActionResult> CreateIdentity(CreateIdentityRequest request, CancellationToken cancellationToken)
    {
        var command = new CreateIdentityCommand
        {
            ClientId = request.ClientId,
            DevicePassword = request.DevicePassword,
            IdentityPublicKey = request.IdentityPublicKey,
            IdentityVersion = request.IdentityVersion,
            SignedChallenge = new SignedChallengeDTO
            {
                Challenge = "-",
                Signature = [0]
            },
            ShouldValidateChallenge = false
        };
```

It shouldn't cause any problems, because in case of the Admin API, the `ShouldValidateChallenge` property of the `CreateIdentityCommand` is set to false anyway.